templates/snippets/page-controls.html

```diff
@@ -1,9 +1,9 @@
 {% load static %}
-<a rel="prev" {% if page_obj.has_previous %}href="?page={{ page_obj.previous_page_number }}"{% else %} class="inactive"{% endif %}>
+<a rel="prev" {% if page_obj.has_previous %}href="?page={{ page_obj.previous_page_number }}"{% endif %} aria-label="previous page">
```

Comment:
A tiny nit, but right now (at least on Chrome) the styling doesn't treat it like a button (i.e. pointer icon) but stays the text selector. I think it should be a CSS style unless I'm missing something.

Reply:
They also aren't appearing in tab order. (I know you'd get this, but flagging so you don't have to chase.)

Reply:
good catch on both of these - I added `cursor:pointer;` and `tabindex=0`. the tab order is still slightly weird because the two sort dropdowns come first, but I think that's not a big deal for keyboard users (and is actually a little more predictable than having the sort dropdowns come "between" the page buttons).